frontend/website/pages/docs/developers/sandbox-container.mdx

```diff
@@ -0,0 +1,47 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Sandbox Container"});
+
+# Sandbox Container
+
+If all you need is a running daemon and some blocks, the sandbox container has everything you need! It uses the same configuration as the testnet, but instead of the community participants ledger it uses a simple ledger with a single demo account.
+
+This can be useful for testing or playing with some of the core features of the protocol.  This node supports multiple accounts, sending transactions between them, and will also support delegating and staking. In fact since it's a single node network, you'll get all the block rewards!
+
+<Alert>
+
+This container will **NOT** connect you to the live network.
+
+</Alert>
+
+## Installation
+
+In order to use the demo container, you'll need to install Docker, the simplest way will be to use [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+After you have Docker installed run the following command to spin up the container.
+
+```
+docker run --publish 3085:3085 -it \
+  codaprotocol/coda-demo:0.0.12-beta-demo-mode-sync-status-a6de071
+```
+
+This command will start a daemon inside the docker container and expose the GraphQL port (3085), which is used for communication with the client, to your computer. This daemon will automatically be running a block producer and snark worker.
+
+### Account details
+
+The container has one account with this public key:
```

Comment:
`coda advanced dump-ledger` returns a public key of `4vsRCVw9P7XQFJWS4FsAsiXRSJyEKiVLCnVw2mRreE7iWout75RvZnm9q46sed2GvBF9Rh972AJrnuhrpPfCDGyhgsJm6kxZGhP5x9CTdty4cpFA8FmxNL8gB2UPTweGnQ1svjTVgUAbb8qB` with a balance of 1000 in addition to that public key with a balance of 100180. This doesn't show up in coda accounts list` more curiosity as my explorer shows 2 accounts.